src/python/zquantum/core/wip/circuits/_compat.py

```diff
@@ -0,0 +1,7 @@
+import zquantum.core.circuit as old_circuit
```

Comment:
Can we name the file `compatiblity` and not `compat` :D ?

Reply:
Sure 🙂 (see 833bf7c)